src/SourceBuild/content/eng/pipelines/ci.yml

```diff
@@ -11,16 +13,30 @@ schedules:
 trigger:
   batch: true
   branches:
-   include:
-   - release/*
-   - internal/release/*
+    include:
+    - main

```

Comment:
With this trigger, the full pipeline will be run on main for every batched CI. This undoes the intent of the dotnet-dotnet-lite pipeline.

It is necessary to separate out main running on the schedule (full build) and main being triggered for every checkin (lite).